packages/editor-ui/src/views/NodeView.vue

```diff
@@ -604,6 +604,11 @@ export default mixins(
 					this.callDebounced('selectAllNodes', 1000);
 				} else if ((e.key === 'c') && (this.isCtrlKeyPressed(e) === true)) {
 					this.callDebounced('copySelectedNodes', 1000);
+					this.$showMessage({
```

Comment:
does not seem like the right place to show this toast. There's a debounce function above that debounces multiple calls. Also what if no nodes are selected.

https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/views/NodeView.vue#L852
seems like a better place..

Reply:
Adding a boolean flag , `isCut` to deduce the operation as the same function is used for cut